spec/requests/users_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    before(:each) do
+      get '/users'
+    end
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'renders the index template' do
+      expect(response).to render_template(:index)
+    end
+    it 'should return placeholder test' do
+      expect(response.body).to include('Placeholder test')
+    end
+    it 'does not render a show template' do
+      expect(response).to_not render_template(:show)
+    end
+  end
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement for this project, and best practice, since we are aiming for 100% test coverage, "`index and show actions for User and Post controllers are tested`", the `show` actions for `User` should be covered with tests.

Reply:
Fixed.